awsiot-optee/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM alpine:3.15 as p11
+
+# A bit ahead of 3.15.0, to include support for secure element control (used by SE05X)
+ENV OPTEE_CLIENT_VER "f4f54e5a76641fda22a49f00294771f948cd4c92"
+
+WORKDIR /src
+
+# build TEE client and headers
+RUN apk add --virtual build-deps git build-base linux-headers
+RUN \
+	git clone https://github.com/OP-TEE/optee_client.git && \
+	cd optee_client && git checkout ${OPTEE_CLIENT_VER} && \
+	wget https://raw.githubusercontent.com/foundriesio/meta-lmp/101b2fcad5d762e70205ae10219eb4e063af59a9/meta-lmp-base/recipes-security/optee/optee-client/0001-FIO-extras-pkcs11-change-UUID-to-avoid-conflict-with.patch && \
+	wget https://raw.githubusercontent.com/foundriesio/meta-lmp/101b2fcad5d762e70205ae10219eb4e063af59a9/meta-lmp-base/recipes-security/optee/optee-client/0001-libckteec-add-support-for-ECDH-derive.patch && \
```

Comment:
You can use the hashes from the v87 tag.

Reply:
i changed this to use the tag instead of the hash. not *quite* the same, but it should be a lot easier to manage and understand moving forward.

Reply:
i retested the new container and it works fine. I accidentally ran the 32bit container on my 64 bit host and that worked also. kind of surprised